devices/ikea/tredansen_block-out_cellul_blind.json

```diff
@@ -1,8 +1,8 @@
 {
   "schema": "devcap1.schema.json",
   "manufacturername": "$MF_IKEA",
-  "modelid": "TREDANSEN block-out cellul blind",
-  "product": "TREDANSEN block-out cellul blind",
+  "modelid": ["TREDANSEN block-out cellul blind", "PRAKTLYSING cellular blind"],
+  "product": "IKEA PRAKTLYSING/TREDANSEN cellular blind",
```

Comment:
Hi, just small correction please don't repeat vendor name in product field.
The `manufacturername` field needs to hold two entries as it's handled as direct mapping to `modelid` field:

```
{
  "manufacturername": ["$MF_IKEA", "$MF_IKEA"],
  "modelid": ["TREDANSEN block-out cellul blind", "PRAKTLYSING cellular blind"],
  "product": "PRAKTLYSING/TREDANSEN cellular blind"
}
```

Reply:
Corrected